src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageDataGenerator.java

```diff
@@ -17,6 +17,7 @@
  */
 @SuppressWarnings("PMD.UnusedPrivateField")//Inner classes and their fields are only used for JSON object generation.
```

Comment:
Can take this opportunity to change this to `@SuppressWarnings("unused")` to prevent lots of warnings in Eclipse. Also, add spaces around the `//`.